### PR TITLE
log_backup: fix initial scan rate limit description

### PR DESCRIPTION
**Pull request by YuJuncen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

这个 PR 修正了对参数 `log-backup.initial-scan-rate-limit` 的默认值的误导性描述。
实际上该配置项参数为 `ReadableSize`，并不是 `MiB`，直接传递 `60` 进去会导致增量扫被限速为 `60B/s`，进而导致增量扫卡住。

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

**Review by BornChanger — APPROVED** [review]

**Comment by joccau** [comment]
LGTM

**Review by joccau — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -2108,8 +2108,8 @@ Raft Engine 相关的配置项。
 
 ### `initial-scan-rate-limit` <span class="version-mark">从 v6.2.0 版本开始引入</span>
 
-+ 日志备份任务在扫描增量数据时的吞吐限流参数。
-+ 默认值：60，即默认限流 60 MB/s
++ 日志备份任务在扫描增量数据时的吞吐限流参数，表示每秒最多从硬盘读出来多少数据。
++ 默认值：60MiB
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:2112]:
```suggestion
+ 日志备份任务在扫描增量数据时的吞吐限流参数，表示每秒最多从硬盘读出的数据量。
+ 默认值：60 MiB
```

**Review by Oreoxmt — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -2108,8 +2108,8 @@ Raft Engine 相关的配置项。
 
 ### `initial-scan-rate-limit` <span class="version-mark">从 v6.2.0 版本开始引入</span>
 
-+ 日志备份任务在扫描增量数据时的吞吐限流参数。
-+ 默认值：60，即默认限流 60 MB/s
++ 日志备份任务在扫描增量数据时的吞吐限流参数，表示每秒最多从硬盘读出的数据量。
++ 默认值：60 MiB
```
Comment (hfxsd) [inline tikv-configuration-file.md:2112]: 一般没有空格？
```suggestion
+ 默认值：60MiB
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -2108,8 +2108,8 @@ Raft Engine 相关的配置项。
 
 ### `initial-scan-rate-limit` <span class="version-mark">从 v6.2.0 版本开始引入</span>
 
-+ 日志备份任务在扫描增量数据时的吞吐限流参数。
-+ 默认值：60，即默认限流 60 MB/s
++ 日志备份任务在扫描增量数据时的吞吐限流参数，表示每秒最多从硬盘读出的数据量。
```
Comment (hfxsd) [inline tikv-configuration-file.md:2111]:
```suggestion
+ 日志备份任务在扫描增量数据时的吞吐限流参数，表示每秒最多从硬盘读出的数据量。注意，如果仅指定数字（如 `60`），则单位为 Byte 而不是 KiB。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**